shell/packages/sandstorm-db/migrations.js

```diff
@@ -672,6 +672,43 @@ function setNewServer() {
   }
 }
 
+function backgroundFillInGrainSizes() {
```

Comment:
Can you move this out to `server/00-startup.js` or somewhere similar?  The fact that it reaches into `globalBackend` makes me believe that it doesn't belong in the `sandstorm-db` module.